src/website/src/pages/ec/components/spotlight/docs/code.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Showcase
+order: 2
+---
+
+import { Playground } from '@ecl/website-components';
+
+## Spotlight
+
+<Playground
+  system="ec"
+  selectedKind="components-spotlight"
+  selectedStory="default"
+></Playground>
+
+## Spotlight width
+
+Spotlight placed inside a grid container (`.ecl-container`) will expand to this container width.
+
+To have a full width spotlight, the recommanded way is to place it outside of the grid container.
```

Comment:
this is very nice, but if we demo also this component with the "full width" hack by default this sentence here will appear for what it is, something that we say for the sake and the respect of our job but that we don't really expect anyone to follow. To me we should demo the component by default without the wrapping container and only wrap it when "full-width" is chosen or even better it would be not to show the full-width toggle at all, the hack and the parameter are there but we don't necessarily have to advertize them.

Reply:
I agree, we should not have the css hack for this component, or at least not in the demo.
But I see one issue: the aspect ratio is different if the component is "full width" or if it is placed alongside a sidebar. Currently this is checked with the full-width class.
So we could keep that class only for that (but then with no possibility to have the css hack), or we would have to introduce a new parameter for the "sidebar" display